client/src/app/site/motions/modules/motion-workflow/components/workflow-detail/workflow-detail.component.ts

```diff
@@ -93,7 +93,7 @@ export class WorkflowDetailComponent extends BaseViewComponent implements OnInit
     /**
      * Determine label colors. Where they should come from is currently now know
      */
-    public labelColors = ['default', 'primary', 'success', 'danger', 'warning'];
+    public labelColors: string[] = ['grey', 'red', 'green', 'lightblue', 'yellow'];
```

Comment:
Are these the new colors? Is the order of this git-diff the same? I can write a migrtion for this, if you are sure not to touch the names again. So @GabrielInTheWorld are you sure that these names are good and won't be changed in the near future?

Reply:
As discussed with @emanuelschuetze these names are the new names for the states.
The names are the color itself, so the user see immediately, which color he selects and this color can be passed to the `mat-chip`s, as it is written.